.github/workflows/im-reusable-finish-build-workflow.yml

```diff
@@ -0,0 +1,162 @@
+        default: 'america/denver'
+      additional-conclusions:
+          description: Valid JSON array of additional conclusion items that should be used to determine if the workflow succeeded or failed
+          required: false
+          type: string
+          default: ''
+      additional-pr-comment-content:
+        description: Additional PR comment content aside from workflow run and next version. 
+        required: false
+        type: string
+      custom-facts-for-team-channel:
+        description: The custom facts that will be included in the post in the team's channel.  By default Workflow, Run, Actor and Version are included.
+        required: false
+        type: string  
+      
+    secrets:
+      MS_TEAMS_URI:
+        description: The URI for the teams channel where a status will be posted.
+        required: true
+      
+
+jobs:
+  finish-build:
+    runs-on: im-linux
+    
+    steps:
+      - uses: im-open/workflow-conclusion@v2.1
+        id: conclusion
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }} # Special per-job token generated by GH for interacting with the repo
+          additional-conclusions: ${{ inputs.additional-conclusions }}
+
+      - name: Configure facts for team's notification channel
+        if: always()
+        id: team-channel-facts
+        uses: actions/github-script@v6
+        with:
+          script: |
+            const rawFacts = process.env.FACTS;
+            console.log(`"${rawFacts}"`);
+            let facts = rawFacts && rawFacts.trim().length > 0 ? JSON.parse(rawFacts) : null;
+            
+            if (!facts || facts.length === 0){
+              console.log(`Custom facts were not provided for the Team's Notification channel, use the default facts:`);
+              facts = [
+                { name: 'Workflow', value: '${{ github.workflow }}'},
+                { name: 'Run', value: '${{ github.run_id }}'},
+                { name: 'Actor', value: '${{ github.actor }}'},
+                { name: 'Version', value: '${{ inputs.next-version }}'}
+              ]
+            }
+            else {
+              console.log(`Custom facts were supplied as an argument, using those in the Team's Notification channel:`);
+            }
+
+            console.log(facts);
+            core.setOutput('facts', facts);
+        env:
+          FACTS: ${{ inputs.custom-facts-for-team-channel }}
+
+      - name: Send status to team's notification channel
+        if: always()
+        continue-on-error: true
+        uses: im-open/post-status-to-teams-action@v1.3
+        with:
+          title: ${{ inputs.title-of-teams-post }}
+          workflow-status: ${{ steps.conclusion.outputs.workflow_conclusion }}
+          workflow-type: Build
+          teams-uri: ${{ secrets.MS_TEAMS_URI }}
+          timezone: ${{ inputs.timezone }}
+          custom-facts: ${{ steps.team-channel-facts.outputs.facts }}
+      
+      - name: Construct PR Comment
+        id: comment
```

Comment:
Can this use Steve's pr-comment action instead so it updates the existing comment?  If not, can this be toggled so teams can add their own comment?